Add FAQs about power outage during CoinJoin (#239)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

[coinfaucet.eu/en/btc-testnet](https://coinfaucet.eu/en/btc-testnet/)
:::

:::details

> ### Does my computer have to stay on during CoinJoin?

Reviewer: ```suggestion
### Does Wasabi have to stay on during CoinJoin?
```

Think saying *Wasabi* is more clear, cause it's not enough that computer is on if Wasabi has been closed.

> Yes your computer needs to stay on during CoinJoins, you cannot be offline and still participate in CoinJoins.

Reviewer: ```suggestion
Yes, Wasabi needs to stay on during CoinJoins, you cannot be offline and still participate in CoinJoins.

A CoinJoin transaction is different from a normal transaction, where you are the only person signing, and requires many people to sign **the same transaction**.
Until all the peers are here, no peer knows what transaction to sign.

Here it's how Wasabi handles different scenarios:

|  | During [input registration phase](FAQ-UseWasabi.md#What-is-happening-in-the-input-registration-phase) | After input registration phase |
|:---:|:---:|:---:|
| You close Wasabi | Your registered coins are automatically dequequed | Wasabi will make you wait until the round finishes |
| Wasabi goes offline | Your registered coins are automatically timed out by the coordinator after 1 minute | Your registered coins will be banned for 24h from particing in another CoinJoin. (This is to prevent [DoS attacks](https://github.com/nopara73/ZeroLink/#d-dos-attack)) |

```

Based on [this](https://www.reddit.com/r/WasabiWallet/comments/deihbr/does_my_computer_have_to_stay_on_during_coinjoin/), [this](https://www.reddit.com/r/WasabiWallet/comments/agi2gk/can_you_close_the_software_while_coinjoin_is/), [this](https://www.reddit.com/r/WasabiWallet/comments/c1wcdz/closing_wasabi_client/) and [this](https://www.reddit.com/r/WasabiWallet/comments/9s0v6m/does_wasabi_need_to_be_running_after_you_setup_a/).

:::

:::details
### What if there's a power outage during CoinJoin? Do I loose my coins?

> No you don't.

Reviewer: ```suggestion
No you don't.
CoinJoins are atomic, they happen or they don't.
```

I read often of users that don't know about the atomic nature of CoinJoin, I think could be useful to add it.

> If your wallet crash or your computer goes offline during CoinJoin you can simply register for a new CoinJoin.

Reviewer: ```suggestion
If your wallet crash or your computer goes offline during CoinJoin you can simply register for a new CoinJoin, look at table [here](FAQ-UseWasabi.md#Does-my-computer-have-to-stay-on-during-CoinJoin) for more info.
```

Author: Link is wrong. Can't commit.
I'll edit myself ;)
